MIP113/MIP113.md

```diff
@@ -244,55 +338,44 @@ The current state of CDs is maintained by the Arbitration Facilitators.
----
-* **5.7.4:** Actual CD income payouts are modified by CD communication metrics for the last 12 months, which requires the CDs to write an explanation for each vote that ties it to the Governance Strategy and their independent strategy, to help the MKR holders that delegated to them to understand the logic they use when executing the chosen Governance Strategies. If the CD actively communicates on less than 95% of all votes, they receive a reduced amount of CD income. The reduction in income is proportionally linear until it reaches 0 CD income at 75% communication activity. If a CD falls below 75% communication activity, it loses qualification for CD income and any CD income rank they may be eligible for is passed on to the next highest CD by amount of delegated MKR.
----
-* **5.7.5:** The 7 PCD slots share a total of 2000 MKR per year amongst them, equivalent to 285.7 MKR per year for each PCD, or 23.8 MKR per month for each PCD.
----
-* **5.7.6:** The 7 RCD slots share a total of 500 MKR per year amongst them, equivalent to 71.4 MKR per year for each RCD, or 5.95 MKR per month for each RCD.
----
-* **5.7.7:** Payout of CD income is done monthly by the Arbitration Facilitators bundling MKR payments into an Executive Vote.
+* **5.4.2:** Reserve Constitutional Delegates (RCDs)
+	* **5.4.2.1:** The top 7 qualifying CDs that are not PCDs by number of combined MKR delegated to their delegate contracts are the RCDs.
+	* **5.4.2.2:** CDs must participate in at least 4 CVC Subcommittee Meetings per month to qualify as RCDs.
+	* **5.4.2.3:** The 7 RCD slots share a total of 500 MKR per year amongst them, equivalent to 71.4 MKR per year for each RCD, or 5.95 MKR per month for each RCD.
+
+* **5.4.3:** CD income payouts are modified by voting activity metrics for votes taking place since they were Recognized, and in the last 12 months.
+	* **5.4.3.1:** If a CD has voted in less than 95% of votes, they receive a reduced amount of CD income.
+	* **5.4.3.2:** The reduction in income is proportionally linear until it reaches 0 CD income at 75% voting activity.
+	* **5.4.3.3:** If a CD falls below 75% voting activity, it receives no CD income.
+
+* **5.4.4:** CD income payouts are modified by communication metrics for votes taking place since they were Recognized, and in the last 12 months.
+	* **5.4.4.1:** To meet the communication requirement for a given vote, CDs must write an explanation of their vote that ties it to their Governance Strategy and their independent strategy.
+	* **5.4.4.2:** If the CD meets the communication requirement on less than 95% of votes, they receive a reduced amount of CD income.
+	* **5.4.4.3:** The reduction in income is proportionally linear until it reaches 0 CD income at 75% communication activity.
+	* **5.4.4.4:** If a CD falls below 75% communication activity, it receives no CD income.
```

Comment:
```suggestion
* **5.4.4:** CD income payouts are adjusted based on communication metrics for votes occurring since they were recognized and within the last 12 months.
	* **5.4.4.1:** If a CD meets the communication requirement for less than 95% of votes, they receive a reduced CD income.
	* **5.4.4.2:** The income reduction is proportionally linear until it reaches 0 CD income at 75% communication activity.
	* **5.4.4.3:** The reduction in income is proportionally linear, reaching 0 CD income at 75% communication activity.
	* **5.4.4.4:** If a CD's communication activity falls below 75%, they receive no CD income.
```